src/types/types.js

```diff
@@ -237,6 +237,7 @@ export type MostRecentWallet = {
 
 export type FioAddress = {
   name: string,
+  bundles: number,
```

Comment:
Should be called `bundledTxs`. Since this is a defined API to `edge-currency-accountbased`, this should be named correctly now as a rename in the future would break the API.

Reply:
currently fioEngine has `bundles` prop for fio address obj. do you want to change it in both places?